src/store/action/participants.ts

```diff
@@ -74,6 +76,16 @@ export const ParticipantActionFactory = {
     showHiddenColumns,
   }),
 
+  /**
+   * Remove a participant from a board session.
+   */
+  setUserBanned: (user: string, userName: string, banned: boolean) => ({
```

Comment:
Does `user` refer to the id? If so, the variable name should be more meaningful, because with `user` I expect the complete user object